File: HappyWayApp.Persistence/Entity/EventMember.cs
using System.Collections.Generic;

namespace HappyWayApp.Persistence.Entity
{
    public class EventMember : EntityBase
    {
        public int Number { get; set; }

        public string Name { get; set; }

        public int Age { get; set; }

        public Sex Sex { get; set; }

        public string PhoneNumber { get; set; }

        public string Email { get; set; }

        public string SocialLink { get; set; }

        public int EventId { get; set; }
        public Event Event { get; set; }

        public List<Like> Liked { get; set; }

        public List<Like> Likes { get; set; }
    }

    public enum Sex
    {
        Male,
        Female
    }
}

File: HappyWayApp/ClientApp/src/app/models/event-member.ts
export class EventMemberModel {
  id: number;
  number: number;
  name: string;
  sex: Sex;
  phoneNumber: string;
  eventId: number;
}

export class EventMemberCardModel {
  member: EventMemberModel;
  targetMembers: CardTargetMember[];
}

export class CardTargetMember {
  id: number;
  number: number;
  liked: boolean;
  name: string;
  phoneNumber: string;
}

export class ResultMemberModel {
  member: EventMemberModel;
  text: string;
}

export class ResultLikedMemberModel {
  number: number;
  name: string;
  phoneNumber: string;
}

export enum Sex {
  Male,
  Female
}
